From 66ccd99c6299ebe70a5549da78063fd7b1e02ea2 Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Wed, 7 Jun 2017 22:28:03 +0300
Subject: [PATCH] Makefile.flags: SURICATTA_HAWKBIT needs librt under older
 glibc

glibc before 2.17 requires linking against librt for clock_gettime(). Fixes
the following build failure:

ld: suricatta/lib.a(server_hawkbit.o): undefined reference to symbol 'clock_gettime@@GLIBC_2.2'

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
Upstream status:
https://groups.google.com/forum/?hl=en#!topic/swupdate/77QWIka27es

 Makefile.flags | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.flags b/Makefile.flags
index 391fc7f82682..8db5dfeacc2f 100644
--- a/Makefile.flags
+++ b/Makefile.flags
@@ -173,6 +173,7 @@ LDLIBS += crypto ssl
 endif
 endif
 ifneq ($(CONFIG_SURICATTA_HAWKBIT),)
+LDLIBS += rt
 ifeq ($(strip $(findstring json-c,$(LDLIBS))),)
 LDLIBS += json-c
 endif
-- 
2.11.0

